src/lib/Libattr/master_sched_attr_def.xml

```diff
@@ -211,7 +211,7 @@
 	<member_at_comp>comp_f</member_at_comp>
 	<member_at_free>free_null</member_at_free>
 	<member_at_action>NULL_FUNC</member_at_action>
-	<member_at_flags><both>NO_USER_SET</both></member_at_flags>
+	<member_at_flags><both>PRIV_READ | ATR_DFLAG_MGWR</both></member_at_flags>
```

Comment:
Why is this different than job_sort_formula (MGR_ONLY_SET)

Reply:
The reference guide says that job_sort_formula_threshold is not user readable while job_sort_formula is, that's why I had to set it differently